unittest_binance_websocket_api.py

```diff
@@ -544,7 +544,7 @@ def test_rest_binance_com_isolated_margin_testnet(self):
         BinanceWebSocketApiRestclient(binance_websocket_api_manager)
         binance_websocket_api_manager.stop_manager_with_all_streams()
 
-    def test_rest_binance_com_futures_testnet(self):
+    def test_rest_binance_com_futures(self):
```

Comment:
I think I found a bug here

Reply:
thanks, the names just need to be unique, then the unittests get executed! Thanks anyway!

Reply:
Unfortunately it was not unique in this case -> see line 552

Reply:
i see, its double! thanks for fixing!